js/apps/admin-ui/src/clients/authorization/policy/Regex.tsx

```diff
@@ -54,6 +55,34 @@ export const Regex = () => {
           {...register("pattern", { required: true })}
         />
       </FormGroup>
+      <FormGroup
+        label={t("targetContextAttributes")}
+        fieldId="pattern"
+        labelIcon={
+          <HelpItem
+            helpText={t("targetContextAttributesHelp")}
+            fieldLabelId="clients:targetContextAttributes"
```

Comment:
You should not add namespaces here.
```suggestion
            fieldLabelId="targetContextAttributes"
```

Reply:
done